2_adventures/001/aniruddhabarapatre/03_day_3/how_old.rb

```diff
@@ -0,0 +1,13 @@
+# If I am 1204 million seconds old, how old am I?
+#
+# doctest: I pass in seconds to get age in years
+# >> seconds_to_year 1204000000
```

Comment:
`1_204_000_000` is easier to read.

Even better if you assign that literal to a variable so that the ... (See next comment)